pubspec.yaml

```
@@ -1,14 +1,15 @@
 name: leak_tracker
-version: 1.0.0-dev
+version: 1.0.0-1.0.dev
```

```suggestion
version: 1.0.0-dev.1.0
```
Semver rules have the prelease with the name before the pre-release version
https://semver.org/#spec-item-9